stake-pool/js/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "stake-pool-js",
+  "version": "0.0.1",
```

We can probably start on version 0.1.0
```suggestion
  "version": "0.1.0",
```